src/components/expandable_section.rs

```diff
@@ -129,21 +132,47 @@ pub fn expandable_section(props: &ExpandableSectionProperties) -> Html {
         class.extend(classes!("pf-m-expanded"));
     }
 
+    let content = html!(
+        <div
+            class="pf-v5-c-expandable-section__content" hidden={!expanded}
+        >{ props.children.clone() }</div>
+    );
+
+    let truncating = matches!(props.variant, ExpandableSectionVariant::Truncate);
```

Comment:
Since `Truncate` doesn't have any fields and implements `PartialEq`, I think `props.variant == ExpandableSectionVariant::Truncate` would be a bit easier to read.